iron-image.html

```diff
@@ -77,6 +79,12 @@
       position: relative;
     }
 
+    #img {
+      display: block;
+      width: var(--iron-image-width, auto);
```

Comment:
+1, added a new issue: #29. this PR will just handle exposing the functionality.